bcda/main.go

```diff
@@ -452,22 +452,22 @@ func createAlphaToken(ttl int, acoSize string) (s string, err error) {
 
 	authProvider := GetAuthProvider()
 
-	params := fmt.Sprintf("{\"clientID\" : \"%s\"}", aco.UUID.String())
-	result, err := authProvider.RegisterClient([]byte(params))
-	if err != nil {
+	params := fmt.Sprintf(`{"clientID" : "%s"}`, aco.UUID.String())
+	result, err := authProvider.RegisterClient(params)
+	if err != nil || result == "" {
 		return "", fmt.Errorf("could not register client for %s (%s) because %s", aco.UUID.String(), aco.Name, err.Error())
```

Comment:
I don't think result can == "" without a non nil error.  I think you need to trust the error returns and not worry about the situation where result == "".

Reply:
Gotcha--looks like a code smell.  Gone in coming commit.